cosmwasm/packages/accounting/src/contract.rs

```diff
@@ -58,6 +58,13 @@ pub enum CommitTransferError {
     MissingWrappedAccount,
 }
 
+/// Validates a transfer without committing it to the on-chain state.  If an error occurs that is
+/// not due to the underlying cosmwasm framework, the returned error will be downcastable to
+/// `CommitTransferErrror`.
```

Comment:
Typo

Reply:
Done.  Also renamed the type to `TransferError` since it is used in 2 different functions now.